scripts/histmakers/mw_lowPU.py

```diff
@@ -37,27 +42,33 @@
                         excl=list(set(args.excludeProcs + ["singlemuon"] if flavor=="e" else ["singleelectron"])),
                         base_path=args.dataPath, 
                         extended = "msht20an3lo" not in args.pdfs,
-                        mode=analysis_label)
+                        mode=analysis_label,
+                        era=args.era,
+                        nanoVersion="v12")
 
 logger = logging.setup_logger(__file__, args.verbose, args.noColorLogger)
 
 for d in datasets: logger.info(f"Dataset {d.name}")
 
-mtw_min=40 # for Wmass (roughly half the boson mass)
+mtw_min = 40 # for Wmass (roughly half the boson mass)
+
+# lepton cuts
+lep_pt_min = 25
 
 # axes used in fakerate calculation
-axis_fakes_pt = hist.axis.Variable(common.get_binning_fakes_pt(args.pt[1], args.pt[2]), name = "pt", underflow=False)
-axis_fakes_eta = hist.axis.Regular(int((args.eta[2]-args.eta[1])*10/2), args.eta[1], args.eta[2], name = "eta", underflow=False, overflow=False)
+axis_fakes_pt = hist.axis.Variable([25, 30, 35, 40, 45, 50, 55], name = "pt", underflow=False, overflow=True)
```

Comment:
I guess this will need to go higher in pt too at some point?

Reply:
Yes, the fakes for the lowPU are to be studied in detail